chord_metadata_service/chord/views_ingest.py

```diff
@@ -153,13 +153,13 @@ def ingest(request):
 
 def _query_and_check_nulls(obj: dict, key: str, transform: Callable = lambda x: x):
     value = obj.get(key, None)
-    return {f"{key}__isnull": True} if value is None else {key: transform(value)}
+    return {f"{key}__isnull": True} if value is None or value == "" else {key: transform(value)}
```

Comment:
I don't understand why this is needed, since `""` isn't the same thing as null so `__isnull` shouldn't work here?

Reply:
Yes, the case was that somewhere it is an empty string in the data (e.g. `"date_of_birth": ""`) and it is not treated as None so it tries to transform.

Reply:
maybe add a flag argument called "convert_empty_str" or something which defaults to false, and if true changes value to None if it's "" in
and then use that flag but only for the specific situation where you need it?

Reply:
is there a drawback of leaving it like this? usually, I wouldn't know if the data has empty strings and where (it can be all properties that accept string).

Reply:
the issue that can happen with this is that if a field can be either None or "" (unlikely but possible...) it will miss instances in the database where "" stored